deps/Makefile

```diff
@@ -263,7 +263,7 @@ endif
 else # NO_GIT
 $2_SRC_DIR = $1-$$($2_BRANCH)
```

Comment:
i suppose it's best to keep them for anyone who does a direct download of the github tarball?